src/style.css

```diff
@@ -0,0 +1,57 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.moyo-logo {
+  height: 40px;
+  width: 40px;
+}
+
+.header-block-links {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.header-list-item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.header-link {
+  font-family: Roboto;
```

Comment:
FYI: Which family will be displaying if google fonts not responding?
We should add additional families, like `Arial` or `sens-serif`.